unit_tests/falco/app/actions/test_configure_interesting_sets.cpp

```diff
@@ -47,7 +47,7 @@ static std::string s_sample_ruleset = "sample-ruleset";
 static std::string s_sample_source = falco_common::syscall_source;
 
 static strset_t s_sample_filters = {
-	"evt.type=connect or evt.type=accept",
+	"evt.type=connect or evt.type=accept or evt.type=accept4 or evt.type=umount2",
```

Comment:
Small touch up on tests to include some of the overlapping syscalls already. Noticed during testing that `init_module` was troublesome ... but it may be a test setup issue, as when I ran the Falco binary with debug log level `init_module` was correctly included everywhere.

As we refactor the ppm sc API one more time with a subsequent libs bump we should include a generic syscalls in the test mock rules as well as this critical test coverage is currently missing. CC @jasondellaluce